Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -52,6 +52,135 @@ function Module:reserveIntValue(max_value)
 	self:allocateInt(max_value)
 end
 
+local function cap(value, limits, cycle)
+	if cycle then
+		if value < limits[1] then
+			return limits[2]
+		end
+		if value > limits[2] then
+			return limits[1]
+		end
+	else
+		if value <= limits[1] then
+			return limits[1]
+		end
+		if value >= limits[2] then
+			return limits[2]
+		end
+	end
+	return value
+end
```

Comment:
I believe `Module.cap` is already defined

Reply:
fixed